schema.sql

```diff
@@ -22,6 +22,9 @@ CREATE TABLE owners (
   PRIMARY KEY (id)
 );
 
+-- updating the schema of owner table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
 CREATE TABLE species (
   id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [ ] Please add the solution you used to decrease the execution time for your queries.
For example, if you added indexes for FOREIGN KEYS(the best way to solve the performance problems) Kindly add them here in the schema file.

Reply:
Thanks for the review I have added the solution used to decrease the execution time of the queries, kindly review it again